public/css/landing.css

```diff
@@ -0,0 +1,83 @@
+body {
+  min-height: 100vh;
+  margin: 0;
+}
+
+.container{
+  width: 90%;
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 5rem;
+}
+  
+.btn{
+  background-color: rgb(180, 180, 180);
+  border-radius: 2rem;
+  color: rgb(27, 27, 27);
+  margin-top: 2rem;
+  padding: 0rem 1.5rem;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: background-color 300ms;
+  position: absolute;
+  right: 30px;
+  bottom: -20px;
+}
+
+img{
```

Comment:
Just add a .tick class to make it more percise.

Having this and adding another img at a later point is a great bug.